.github/workflows/build.yml

```diff
@@ -52,10 +54,10 @@ jobs:
         run: |
           if grep -q authToken ~/.config/cachix/cachix.dhall; then
             echo "Cachix token is present"
-            cachix watch-exec nixpkgs-terraform nix -- flake check --max-jobs 2
+            cachix watch-exec nixpkgs-terraform nix -- flake check
```

Comment:
@oscar-izval as we would be building fewer versions, I believe we could be more flexible about the number of derivations to build at the same time.